ethsigner/core/build.gradle

```diff
@@ -29,23 +29,20 @@ jar {
 dependencies {
 
   implementation project(':ethsigner:signing-api')
-
   implementation 'com.google.guava:guava'
-
   implementation 'org.apache.logging.log4j:log4j-api'
-  runtime 'org.apache.logging.log4j:log4j-core'
-  runtime 'org.apache.logging.log4j:log4j-slf4j-impl'
-
   implementation 'org.web3j:crypto'
   implementation 'org.web3j:besu'
-
   implementation 'io.vertx:vertx-core'
   implementation 'io.vertx:vertx-web'
   implementation 'io.vertx:vertx-web-client'
-
-  runtime 'org.bouncycastle:bcpkix-jdk15on'
   implementation 'org.apache.tuweni:tuweni-net'
 
+  api 'org.apache.logging.log4j:log4j-core'
```

Comment:
Don't add the `core` as an `api`. If you need log4j as an `api` instead use `api 'org.apache.logging.log4j:log4j-api'`